proposals/lip-0061.md

```diff
@@ -112,14 +112,14 @@ The certificate generation specified in this LIP therefore generates an aggregat
 
 Note that if possible, also aggregate commits for blocks that are not at the end of the round are created and added to blocks so that a certificate is created as soon as a block is finalized and it is not required to wait for a change of BFT parameters.
 
-## Minimum Certificate Height
+### Minimum Certificate Height
 
 During the bootstrap period for Lisk Core v4, as defined in [LIP 63][lip-0063], the 101 fixed initial validators will not have valid registered BLS keys from the beginning. 
```

Comment:
I know it was not introduced here, but since we are cleaning up w.r.t. bootstrap period, we could set the link more specifically to https://github.com/LiskHQ/lips/blob/main/proposals/lip-0063.md#bootstrap-period.

Reply:
Good point. Done.